core/src/com/cloud/agent/api/storage/ListTemplateCommand.java

```diff
@@ -43,6 +49,10 @@ public DataStoreTO getDataStore() {
         return store;
     }
 
+    public String getNfsVersion() {
+        return nfsVersion;
+    }
+
```

Comment:
This looks like a useful change to the storage API, but why is it necessary to implement OSPF?  Seems like it should be in a separate PR.